recipes/datacompy/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "datacompy" %}
+{% set version = "0.6.0" %}
+
+ package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+ source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ae266c96bd96e090f3daa91668094bab487fdceff97c65c8f05324c7dabc0358
+
+ build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

These are arguments are not needed anymore (but included by default).
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```